### PR TITLE
Edit documentation

### PR DESCRIPTION
**Pull request by YanchoK**
Closes #{issue number}

## Screenshots:

Before|After
---|---
Paste screenshot|Paste screenshot

### New environment variables:

- [ ] `NEW_ENV_VAR`: env var details

### New or updated dependencies:

Dependency name|Previous version|Updated version|Details
---|---|---|---
`dependency/name`|`v1.0.0`|`v2.0.0`|

### CODE REVIEW

**Review by ani-kalpachka — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,7 +22,12 @@ Table of contents:
 
 ### Setup local dev environment
 
-To run and develop the module NodeJS 16 is required. In this section 2 ways of configuring a development environment are described.
+To run and develop the module NodeJS 16.13.0 is required.
```
Comment (imilchev) [inline CONTRIBUTING.md:25]: Is it really that exactly 16.13.0 is required? I would expect that it works with any v16?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -44,14 +49,11 @@ cd frontend
...
+# Symlink dev environment on Windows (CMD. run as admin)
 mklink .env.local .env.local.example
 
-# Install dependencies
-yarn
-```
-
-### NOTE! For existing clones update github remote to podkrepi-bg/frontend
+cal .env.local.example
+ng clones update github remote to podkrepi-bg/frontend
```
Comment (imilchev) [inline CONTRIBUTING.md:56]: Isn't `ng` an Angular command? Why is it needed for this project where we aren't using Angular at all?

File: README.md [hunk 3]
```diff
@@ -49,11 +49,22 @@ yarn dev
...
 
-In order to use the frontend together with the API for it you need to clone and start the backend project.
+In order to use the frontend together with the API for it you need to clone and start the backend project or you can use the API in production https://dev.podkrepi.bg/api/v1.
+
+To start the backend project follow API instructions at <https://github.com/podkrepi-bg/api#setup-development-environment>
 
-Follow API instructions at <https://github.com/podkrepi-bg/api#setup-development-environment>
+To use the API in production change .env.local.example
+```
+API_URL=http://podkrepi.bg/api/v1
```
Comment (imilchev) [inline README.md:65]: I would suggest to absolutely NOT use the production URLs for local dev or testing.

File: README.md [hunk 4]
```diff
@@ -49,11 +49,22 @@ yarn dev
...
+IMAGE_HOST=podkrepi.bg
```
Comment (imilchev) [inline README.md:66]: I think that this should always be localhost if you are running the frontend locally.

**Review by imilchev — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (kachar) [reply CONTRIBUTING.md:56]: We also miss the `yarn install` command

**Comment by kachar** [comment]
Closing the PR until the changes are reflected. Feel free to reopen it

**Pull request closed without merging**